.github/workflows/ci_v_benchmark.yml

```diff
@@ -0,0 +1,44 @@
+name: vlang benchmarks
+
+on:
+  push:
+    paths-ignore:
+      - "**.md"
+  pull_request:
+    paths-ignore:
+      - "**.md"
+
+jobs:
+  run:
+    name: Run
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up GCC
+      uses: egor-tensin/setup-gcc@v1
+      with:
+        version: latest
+        platform: x64
+    - name: Setup cmake
+      uses: jwlawson/actions-setup-cmake@v1.12
+      with:
+        cmake-version: '3.16.x'
```

Comment:
same - afaik, cmake is installed by default

Reply:
From https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md :
```
CMake 3.23.0
GNU C++ 9.4.0, 10.3.0
```

Reply:
Removed, we will give a try :) Thanks!